src/components/widgets-settings/bolt-account/bolt-account.tmpl.html

```diff
@@ -0,0 +1,5 @@
+<h5 ng-if="showLabel">{{label}}</h5>
+<div class="input-group settings select-account">
+	<select ng-model="parentWidget.selectedAccount" ng-change="onAccountSelected()" ng-options="formatLabel(account) for account in parentWidget.accountList track by account.name" class="form-control">
```

Comment:
You shouldn't use `parentWidget` if this can be avoided: pass the `accountList` to your directive as an argument instead, this will keep the concerns separated